### PR TITLE
boost review requirements

### PR DESCRIPTION
**Pull request by vinniefalco**
File moving and renaming, copyright change

### CODE REVIEW

File: src/beast/include/beast/streambuf.hpp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) 2013-2016 Vinnie Falco (vinnie dot falco at gmail dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#ifndef BEAST_STREAMBUF_HPP
+#define BEAST__STREAMBUF_HPP
```
Comment (seelabs) [inline src/beast/include/beast/streambuf.hpp]: Wrong header guard (extra underscore)

**Comment by miguelportilla** [comment]
Extra underscore:
https://github.com/vinniefalco/rippled/blob/server/src/beast/include/beast/streambuf.hpp#L9

**Comment by seelabs** [comment]
:+1:

**Comment by miguelportilla** [comment]
:+1:

**Comment by vinniefalco** [comment]
Finally got this monster in.

**Pull request closed without merging**